Assets/UnZENity-VR/Scripts/Components/UI/VRDialog.cs

```diff
@@ -49,6 +49,12 @@ public void ShowDialog(GameObject npcGo)
             hoverElements.AddRange(_dialogItems.Select(i => i.GetComponentInChildren<TMP_Text>().gameObject).ToList());
             _uiEventsHandler.SetElementsToHover(hoverElements, true);
             
+            StartCoroutine(ShowDialogWithDelay());
+        }
+
+        private System.Collections.IEnumerator ShowDialogWithDelay()
+        {
+            yield return new WaitForEndOfFrame();
```

Comment:
Smart. This is the one, that the dialog rotation isn't shown in game?
Please add a comment why this call is done.